modules/ztp-provisioning-lvm-storage.adoc

```diff
@@ -37,13 +39,31 @@ You use {lvms} to persist event subscriptions when you use PTP events or bare-me
     channel: stable-{product-version}
   policyName: subscription-policies
 ----
++
+[NOTE]
+====
+The Storage LVMO subscription is deprecated. In future releases of {product-title}, the storage LVMO subscription will not be available. Instead, you must use the Storage LVMS subscription. 
```

Comment:
```suggestion
The Storage LVMO subscription is deprecated and will not be available in future releases of {product-title}. Instead, you must use the Storage LVMS subscription. 
```

There's nothing wrong with what you have, I was just looking at reducing word count. Fine either way--just noting.